table.csv

```diff
@@ -440,6 +440,10 @@ zcash-tx,             Zcash Tx,                                         0xc1
 stellar-block,        Stellar Block,                                    0xd0
 stellar-tx,           Stellar Tx,                                       0xd1
 
+leofcoin-block,       Leofcoin Block,                                   0x0A
+leofcoin-tx,          Leofcoin Transaction,                             0x54
+leofcoin-pr,          Leofcoin Peer Reputation,                         0x52
```

Comment:
I would prefer them being above `0x80` thus being in the two byte varint rage same as all other blockchains.

Reply:
Sure, do you have any preference?

Reply:
Not really, anything over 0x80 would be fine.